src/elements/Slider.tsx

```diff
@@ -0,0 +1,121 @@
+import { Range } from "rc-slider"
+import React from "react"
+import styled from "styled-components"
+import {
+  BorderProps,
+  borders,
+  space as styledSpace,
+  SpaceProps,
+} from "styled-system"
+import { color, space } from "../helpers"
+
+export interface SliderProps extends BorderProps, SpaceProps {
+  className?: string
+  allowCross: boolean
+  disabled?: boolean
+  min: number
+  max: number
+  step: number
+  defaultValue: number[]
+  onAfterChange?: (min_max: [number, number]) => void
```

Comment:
`min_max` -> `minMax` ?

Reply:
yeah, agreed. I was hoping I could do `([min, max]: [number, number]) => void` but that doesn't work so I named it that way